SWIPs/swip-message_to_honey.md

```diff
@@ -13,28 +13,33 @@ Created: 31-07-2019
+
+An on-chain price oracle, managed by a multi-signature wallet of Swarm stakeholders, provides a clean way for updating relative prices; the Swarm source code will never require an update but instead references the address of the oracle which updates its quoted prices at predictable times.
 This SWIP is part of a series of SWIPs (but can be implemented on it's own). To see the full picture, please refer to [swip-multiple_payment_processing_support](./swip-multiple_payment_processing_support.md), [swip-honey_to_money](./swip-honey_to_money.md) and the diagram below:
 
 ![SWIP_Diagrams.svg](./../assets/swip-message_to_honey/SWIP_Diagrams.svg)
 
 ## Abstract
 <!--A short (~200 word) description of the technical issue being addressed.-->
-To enable changing the relative prices of Swarm messages for all nodes simultaneously, Swarm will provide a `msgToHoney` payment oracle, initially managed by Swarm stakeholders and developers. The `msgToHoney` oracle receives a message type and returns a price + validUntil tuple. The `msgToHoney` oracle lives on the Ethereum blockchain and is managed by a multi-signature of Swarm stakeholders and developers. Once nodes receive a message to account for, they will query their local database on whether they have a valid price for this message. If this is the case, the price will be applied by the node, if this is not the case, the nodes will query the `msgToHoney` oracle. The oracle provides a new price before the old price expires, to allow nodes to pre-load new prices before old ones expire. If the oracle fails to submit a new price, while the old price is expired, nodes will continue querying the `msgToHoney` oracle at a set interval and apply the most-recent price known to them. 
+To enable changing the relative prices of Swarm messages for all nodes simultaneously, Swarm will provide a `MsgToHoney` payment oracle, initially managed by Swarm stakeholders and developers. The `MsgToHoney` oracle receives a message type and returns a `(price,validUntil)` tuple. The `MsgToHoney` oracle lives on the Ethereum blockchain and is managed by a multi-signature contract of Swarm stakeholders and developers. Once nodes receive a message to account for, they will query their local cache for a valid price for this message. If valid, the price will be applied by the node, if not, the nodes will query the `MsgToHoney` oracle. The oracle provides a new price before the old price expires, to allow nodes to pre-load new prices before old ones expire. If the oracle fails to submit a new price, while the old price is expired, nodes will continue querying the `MsgToHoney` oracle at a set interval and apply the most-recent price known to them. 
```

Comment:
Same governance issue

Reply:
I propose to include governance as a general pointer.